README.md

```diff
@@ -48,9 +48,9 @@ Transaction received are formatted as defined below
 
 1. Store the received transaction using the SQLite database provided in `utils/database.ts`
 2. Compute the `balance` per bank account & store it.
-3. Compute `nextBalance` per bank account & store it.
 
 ### Optional goals
 
-4. Compute a `remainingPaymentCardAmount` for the bank account with a default value **10000 EUR with no limit of time**.
-   We should be able to modify the default value for a given bank account later.
+3. Compute `nextBalance` per bank account & store it.
+<!-- 4. Compute a `remainingPaymentCardAmount` for the bank account with a default value **10000 EUR with no limit of time**.
```

Comment:
What about totally removing this one?

Reply:
Yes we can. It was in case we want to put it back someday but it will probably never happen